Podcast/SearchUser.swift

```diff
@@ -11,18 +11,20 @@ import SwiftyJSON
 
 class SearchUser: NSObject {
```

Comment:
do we need a search user anymore? I have been deleting the partial models

Reply:
I thought the eventual plan was to go back to the partial model, so for now leave in partials and subclass?

Reply:
I was thinking for now delete, since its extra files and we really don't use them in terms of fail-proof code. Shouldn't be too hard to delete this.

Reply:
Alright.